recipes/cad1/task2/baseline/audio_manager.py

```diff
@@ -0,0 +1,90 @@
+"""A utility class for managing audio files."""
+# pylint: disable=import-error
+
+import logging
+from pathlib import Path
+from typing import Dict, Tuple
+
+import numpy as np
+from scipy.io import wavfile
+
+logger = logging.getLogger(__name__)
+
+
+class AudioManager:
+    """A utility class for managing audio files."""
+
+    def __init__(
+        self,
+        sample_rate: int = 44100,
+        output_audio_path: str = "",
```

Comment:
In the `evaluate()` function you are passing in a `Path()` object. This should perhaps therefore have the following typehint...

```suggestion
        output_audio_path: Union[str, Path] = "",
```